### DISCUSSION TITLE
Open Source Certification - what the...!?!?!@#%^

### DISCUSSION THREAD

**Discussion by m3tomlins** [m3tomlins/performacology#2] [General]
Well? What do you think it could be?

**Comment by ericproegler** [comment]
An easy to administer, asyncronous "exam" could consist of 
1. A given data set of measurements and tests - diagnosis
2. A series of essay questions to describe learnings
3. A completion record of some kind for content - this person read/watched/listened to x body of knowledge

Review/grading is hard. For Black Box Software Testing, we use cohorts. They review each others' work, and then a group of volunteers also provide feedback. It's really pass/fail, and almost any amount of discernable effort is sufficient to pass

**Reply by m3tomlins** [reply]
I appreciate your sharing BBST experience on grading. I wonder if we could set a minimum # of reviewers required on a pull request into the repo to drive some consensus?

1 - I write up a submission for review and acceptance and submit a PR.
2 - Review, comments and questions/answers are transparent on the PR thread.
3 - achieving enough upvotes/reviews would approve the PR and get merged

Hmmm...

**Comment by ericproegler** [comment]
I used to give this talk - [show graphs and ask people to tell me what was happening](https://www.slideshare.net/eproegler/interpreting-performance-test-results). Some seem quaint now, but as a pattern, this is not bad. What are the classic curves?

**Comment by ericproegler** [comment]
A model to encourage/invite contribution to a body of knowledge is another way for people to demonstrate and participate as a learning activity

**Reply by m3tomlins** [reply]
Considering the historic diversity of topics from all the WOPRs'z...we could have some structured categories for submissions?

We must be cognizant of the submitters' experience that "my submission needs to compete with all the others" and the biases that we would incur therein. This could be a big stumbling block to avoid.

How would open-source coding/contribution help to solve those biases? What things have we corrected/improved in the open-source coding world that we should inherit?

**Comment by ericproegler** [comment]
A report on some real work, with some graphs and analysis?

**Reply by m3tomlins** [reply]
It's not coincidence that the inspiration for this performacology thingy came during the week of WOPR29 in Uruguay where we had some amazing experience reporting! Much like there is some prescriptive guidance on how to author an experience report, could we have a similar structure or guidance for submissions to this repo? 

And of course, there is more than just your submission that goes into making an excellent ER - it's the structure of how it's presented and the interactions it inspires.

**Comment by hohenner** [comment]
I like the idea of a certification being something other than a test that you can study for, and pass by only giving the “correct” answers.

**Reply by ericproegler** [reply]
Worked on this: https://associationforsoftwaretesting.org/principles-and-positions/certifications/
